osu.Framework.Font/Graphics/Sprites/KaraokeSpriteText_DrawNode.cs

```diff
@@ -66,7 +66,7 @@ protected override long GetDrawVersion()
             return base.GetDrawVersion();
         }
 
-        public List<DrawNode> Children
+        public List<DrawNode>? Children
```

Comment:
~~Not very sure about should let it become nullable?~~
~~Because I remember `InternalChildren` in the `CompositeDrawable` will not be nullable if apply the `NRT`. so the same things should apply in the draw node i guess?~~

[Update]
peppy said it's nullable, so it's nullable 
https://github.com/ppy/osu-framework/commit/e9f08ad8874a86b08dcea54c63f0a56acf993435